static/katex.less

```diff
@@ -581,6 +581,41 @@
         position: relative;   // ditto
         overflow: hidden;     // This line applies to all browsers.
     }
+	
+	.halfarrow-left {
+		position: absolute;
+		left: 0;
+		width: 50.1%;
+		overflow: hidden;
+	}
+
+	.halfarrow-right {
+		position: absolute;
+		right: 0;
+		width: 50%;
+		overflow: hidden;
+	}
+
+	.brace-left {
+		position: absolute;
+		left: 0;
+		width: 25.1%;
```

Comment:
The extra 0.1% here and in brace-right is to overlap the center part of the brace, right?  Did certain browsers exhibit a gap without the 0.1%?

Reply:
Yes, I have seen browsers exhibit a one screen-pixel gap when I tried to set widths to exactly 50%. It happened occasionally, not frequently. It happened in multiple browsers, not any particular browser. I think the behavior is probably due to floating point rounding error.

Now to be precise, I observed those gaps when I was using SVGs in background images. I suppose that the current inline SVGs, written directly into the DOM, might act differently. But I feel a lot more secure knowing that I have that 0.1% overlap.

Reply:
FWIW, this is a general rendering/antialiasing problem with most SVG renderers I've encountered (including Chrome): abutting features generally leave tiny gaps.  So definitely approve of the slight overlap (assuming 0.1% is never too big?).